tests/test_postprocess_predictions.py

```diff
@@ -59,3 +69,157 @@ def test_clean_vectordata(tmpdir):
     result_gdf = gfo.read_file(geoms_simpl_filepath)
 
     assert len(result_gdf) == 616
+
+
+def create_projects_dir(tmp_path: Path) -> Path:
+    testproject_dir = tmp_path / "orthoseg_test_postprocess"
+    project_dir = testproject_dir / "footballfields"
+
+    shutil.rmtree(path=testproject_dir, ignore_errors=True)
+    project_dir.mkdir(parents=True, exist_ok=True)
+    shutil.copyfile(
+        src=test_helper.sampleprojects_dir / "project_template/projectfile.ini",
+        dst=project_dir / "footballfields.ini",
+    )
+    shutil.copyfile(
+        src=test_helper.sampleprojects_dir / "imagelayers.ini",
+        dst=testproject_dir / "imagelayers.ini",
+    )
+    return project_dir
+
+
+def create_prediction_file(output_vector_dir: Path) -> Path:
+    imagelayer = "BEFL-2019"
+    prediction_dir = output_vector_dir / imagelayer
+    prediction_dir.mkdir(parents=True, exist_ok=True)
+    output_vector_path = prediction_dir / f"footballfields_01_201_{imagelayer}.gpkg"
+
+    footballfields = {
+        "classname": [
+            "footballfields",
+            "footballfields",
+            "footballfields",
+            "footballfields",
+            "footballfields",
+        ],
+        "geometry": [
+            shapely.wkt.loads(
+                "MULTIPOLYGON (((175056.375 176371.125, 175056.375 176370.875, 175056.625 176370.875, 175056.625 176371.125, 175056.375 176371.125)))"  # noqa: E501
+            ),
+            shapely.wkt.loads(
+                "MULTIPOLYGON (((175054.875 176370.875, 175054.875 176370.625, 175055.375 176370.625, 175055.375 176370.875, 175054.875 176370.875)))"  # noqa: E501
+            ),
+            shapely.wkt.loads(
+                "MULTIPOLYGON (((175047.125 176370.875, 175047.125 176370.125, 175047.375 176370.625, 175047.125 176370.875)))"  # noqa: E501
+            ),
+            shapely.wkt.loads(
+                "MULTIPOLYGON (((175046.625 176370.125, 175046.625 176369.875, 175046.875 176369.875, 175046.875 176370.125, 175046.625 176370.125)))"  # noqa: E501
+            ),
+            shapely.wkt.loads(
+                "MULTIPOLYGON (((175054.625 176369.625, 175054.625 176368.875, 175055.625 176369.375, 175054.625 176369.625)))"  # noqa: E501
+            ),
+        ],
+    }
+    footballfields_gdf = gpd.GeoDataFrame(footballfields)
+    footballfields_df = footballfields_gdf.set_geometry("geometry")
+    gfo.to_file(gdf=footballfields_df, path=output_vector_path)
```

Comment:
```suggestion
    footballfields_gdf = gpd.GeoDataFrame(geometry=footballfields)
    gfo.to_file(gdf=footballfields_gdf, path=output_vector_path)
```
I think this should be OK as well.